### PR TITLE
[14.0][IMP] add description field to bookings

### PR DESCRIPTION
**Pull request by huguesdk**
add a description text field to resource bookings to allow the organizer to add more information.

### CODE REVIEW

File: resource_booking/models/resource_booking.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ class ResourceBooking(models.Model):
         ),
     )
     name = fields.Char(index=True, help="Leave empty to autogenerate a booking name.")
+    description = fields.Text("Description")
```
Comment (pedrobaeza) [inline resource_booking/models/resource_booking.py]: Why not HTML to have rich text format?

Comment (cathLemb) [reply resource_booking/models/resource_booking.py]: The client didn't specify, so it's ok for me to change to html.

Comment (huguesdk) [reply resource_booking/models/resource_booking.py]: done.

**Review by pedrobaeza — APPROVED** [review]

**Review by carmenbianca — APPROVED** [review]
might be good practice to add a changelog. otherwise it's a good simple change :) lgtm

**Comment by pedrobaeza** [comment]
I don't like changelog, as it's double work, and inconsistencies between both "source of truth". I prefer very good commit messages instead.

/ocabot merge minor

**Comment by huguesdk** [comment]
thanks, @pedrobaeza!

**Pull request merged**